qemu/tests/cfg/numa.cfg

```diff
@@ -11,22 +15,21 @@
             only Linux
             type = numa_consistency
             start_vm = no
-            threshold = 0.05
-            ppc64,ppc64le:
-                threshold = 0.15
+            mem_fixed = 4096
+            backend_mem = "memory-backend-ram"
+            size_mem = ${mem_fixed}M
+            use_mem = no
```

Comment:
It is useless in this case as no mem device.

Reply:
Totally true, updated :grinning: